uwmweb/uwmcms
# UWM-RAR: Refinement of GDPR 550680338

## Description:
Use GDPR's theme styling and scripts with relying on EU Cookie Module's. This allows the banner to use the Module's settings without script overhead to override scripts. The scripting and styling are designed to work with Jump to Top and Share Buttons.

## Testing:

- Import a current version of Rar's DB from Acquia.
- Run a configuration import to bring in an updated setting for EU Cookie Module.
- Go to /admin/config/system/eu-cookie-compliance and verify banner is enabled and permissions are set for anonymous.
- Check what the Cookie's name will be.
- Go to /admin/appearance/settings/rar and verify "User Interface Components" are checked on.
- Using incognito mode, go to the homepage and verify banner displays. 
-  Click "Accept" on a banner and verify the cookie set.
